app/views/api/v3/analytics/user_analytics/_progress_monthly_report.html.erb

```diff
@@ -46,4 +46,18 @@
       <%= render(Reports::MonthlyProgressComponentV1.new(dimension, service: @service, current_user: @current_user)) %>
     <% end %>
   </div>
+
+  <% if CountryConfig.current_country?("India") %>
```

Comment:
Instead of country config directly here, you can make it a helper in the controller.